api/v1beta1/cpu_pressure.go

```diff
@@ -12,7 +12,10 @@ import (
 
 // CPUPressureSpec represents a cpu pressure disruption
 type CPUPressureSpec struct {
-	Count *intstr.IntOrString `json:"count,omitempty"` // number of cores to target, either an integer form or a percentage form appended with a %
+	// Count represents the number of cores to target
+	// either an integer form or a percentage form appended with a %
+	// if empty, it will be considered to be 100%
+	Count *intstr.IntOrString `json:"count,omitempty"`
```

Comment:
what was the behaviour before, when we didn't specify a count?

Reply:
The exact same one, if count was unspecified we basically did a 100% disruption.

Originally the disruption spec was countless.

Then, we introduced the count that had various meanings.